tests/nest_tests/nest_integration_test.py

```diff
@@ -108,7 +115,10 @@ def test_nest_integration(self):
                               {"tau_syn_rise_E": 2., "tau_syn_decay_E": 10., "tau_syn_rise_I": 2.,
                                "tau_syn_decay_I": 10.}))  # XXX: TODO: does not work yet when tau_rise = tau_fall (numerical singularity occurs in the propagators)
 
-        neuron_models.append(("izhikevich", "izhikevich_nestml", 1E-3, 1))        # large tolerance because NEST Simulator model does not use GSL solver, but simple forward Euler
+        if nest_version.startswith("v2"):
+            neuron_models.append(("izhikevich", "izhikevich_nestml", 1E-3, 1, {}, {}, {"V_m": -70., "U_m": .2 * -70.}))        # large tolerance because NEST Simulator model does not use GSL solver, but simple forward Euler
```

Comment:
Why are the values different for different NEST versions?

Reply:
The initial values of the Izhikevich neuron have changed from NEST 2 to NEST 3. This line is to make them consistent between both NEST versions and NESTML model defaults.